client/reader/search-stream/search-follow-button.jsx

```diff
@@ -0,0 +1,82 @@
+import { Gridicon } from '@automattic/components';
+import { localize } from 'i18n-calypso';
+import { some } from 'lodash';
+import PropTypes from 'prop-types';
+import { Component } from 'react';
+import { connect } from 'react-redux';
+import { resemblesUrl, withoutHttp, addSchemeIfMissing } from 'calypso/lib/url';
+import ReaderFollowFeedIcon from 'calypso/reader/components/icons/follow-feed-icon';
+import ReaderFollowingFeedIcon from 'calypso/reader/components/icons/following-feed-icon';
+import FollowButton from 'calypso/reader/follow-button';
+import { SEARCH_RESULTS_URL_INPUT } from 'calypso/reader/follow-sources';
+import { getReaderAliasedFollowFeedUrl } from 'calypso/state/reader/follows/selectors';
+import { commonExtensions } from 'calypso/state/reader/follows/selectors/get-reader-aliased-follow-feed-url';
+import './style.scss';
+
+class SearchFollowButton extends Component {
+	static propTypes = {
+		query: PropTypes.string,
+		feed: PropTypes.object,
+	};
+
+	render() {
+		const { query, translate, readerAliasedFollowFeedUrl, feed } = this.props;
+		let isPotentialFeedUrl = false;
+		if ( resemblesUrl( query ) ) {
+			const parsedUrl = new URL( query );
```

Comment:
this throws an error if query has a URL without a protocol e.g. "`wordpress.com`"

Reply:
Should be fixed now